elkscmd/Makefile

```diff
@@ -67,6 +18,6 @@ install:
 	$(MAKE) -f Make.install "CONFIG=$(TOPDIR)/.config"
 
 clean:
-	for DIR in $(SUBDIRS); do $(MAKE) -C $$DIR clean || exit 1; done
+	for DIR in `sed -e '/^#/d' -e 's/\/.*//' Applications | sort | uniq`; do $(MAKE) -C $$DIR clean || exit 1; done
```

Comment:
Doesn't this line and line 12 make compilation and cleaning depending on the contents of the Applications file? That should not be the case, the script should auto-iterate through subdirectories independently.